testdata/serviceconfig.yml

```diff
@@ -7,6 +7,23 @@ rules:
   aspects:
   - kind: quotas
     params:
+  - kind: metrics
+    adapter: "prometheus"
+    params:
+      metrics:
+      - descriptor: "request_count"
+        value: "1"
```

Comment:
From the expression languages standpoint `"2"` will be evaluated to a string and `2` will be evaluated to an `int64` and looks like for correct operation you __want__ it to be `int64`